src/enclave_proc/resource_manager.rs

```diff
@@ -53,6 +53,9 @@ type UnpackedHandle = (u64, u64, u64, Vec<u32>, u64, u64, EnclaveState);
 /// The bit indicating if an enclave has been launched in debug mode.
 pub const NE_ENCLAVE_DEBUG_MODE: u64 = 0x1;
 
+/// Constant number used for computing the lower memory limit.
+const NE_EIF_MEM_THRESHOLD: u64 = 4;
```

Comment:
ENCLAVE_MEMORY_EIF_SIZE_RATIO would be more suggestive.

Reply:
Renamed in the latest revision.